n6001/hardware/ofs_n6001/build/scripts/compile_script.tcl

```diff
@@ -7,10 +7,6 @@ post_message "Running compile_script.tcl script"
 # get flow type (from quartus(args) variable)
 set flow [lindex $quartus(args) 0]
 
-puts [exec cp -rf ipss ../..]
```

Comment:
This was the hack from last week which was rendered useless by further FIM-changes since then.